sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp

```diff
@@ -1067,8 +1076,9 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
         std::string ServiceEndpoint;
         std::string ShareName;
         Nullable<std::string> ShareSnapshot;
+        Nullable<bool> Encoded;
         std::string DirectoryPath;
```

Comment:
Is this field `DirectoryPath` visible to end users? Do we need to decode it in convenience layer?

Reply:
It's visible to end users  and I will decode it in convenience layer.